src/frontend/components/LoginBanner.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import loginBanner from './loginBanner'
```

Comment:
Is this the styles for this component? If so, please name the file `style.scss`.